### ISSUE TITLE
neon is a bit bright

### ISSUE THREAD

**Issue by gulp78** [Roblox/future-is-bright#16]
i went onto my night time test game with neon lighting and it is a bit bright for what i use it for [https://images.discordapp.net/attachments/336864753743822870/338511315112951818/RobloxScreenShot20170723_123642632.png?width=946&height=470](url)

**Comment by Sparkypen7** [comment]
![image](https://user-images.githubusercontent.com/11379504/28496526-930fe57a-6f3b-11e7-96ae-91bfea25eb21.png)

Are those pixel-like figures around the train lights?

**Comment by gulp78** [comment]
yeah, i didn't change it to shadowmap but neon is the same on every single one

**Comment by TheSink** [comment]
Yeah. It's almost like they increased the intensity of the neon/made it more blurry, I don't like how it looks though at all. It only looks good for a very select few reasons, and is waaay too bright.

**Comment by zeux** [comment]
Yup, neon needs more tuning.

**Comment by jackyloz** [comment]
I personally think that if Neon is enabled there should be a separate property to control it's brightness. I feel as if that level of brightness should be available as one of the higher values if so. I don't want to have to resort to using transparency as it plain just looks weird.
